lib/cartopy/mpl/geoaxes.py

```diff
@@ -1443,7 +1443,7 @@ def _pcolormesh_patched(self, *args, **kwargs):
         collection.set_alpha(alpha)
         collection.set_array(C)
         if norm is not None:
-            assert(isinstance(norm, mcolors.Normalize))
+            assert (isinstance(norm, mcolors.Normalize))
```

Comment:
I don't think this line needs the outer brackets at all. There's no line continuation or explicit operation order going on here...

Reply:
I actually didn't touch that... I'm confused...

Reply:
Has your editor gained sentience??

Reply:
I think it might have done.  It's certainly started being a bit stroppy and irrational lately...